### DISCUSSION TITLE
Video: How to get started with testing in Laravel Livewire

### DISCUSSION THREAD

**Discussion by jonneroelofs** [livewire/livewire#4276] [2. Show and tell]
I made a youtube video for people who are familiar with Livewire but don't know how they can write tests for it. Hope it helps someone.

[How to get started with testing in Laravel Livewire](https://youtu.be/2ReWoRV4saY)

**Comment by joshhanley** [comment]
Great video @jonneroelofs! 😁

**Reply by jonneroelofs** [reply]
Thanks a lot!
